src/TBuffer.cpp

```diff
@@ -1791,7 +1791,7 @@ void TBuffer::translateToPlainText(std::string& incoming, const bool isFromServe
                     mIgnoreTag = false;
                     mSkip.clear();
                     ch = '&';
-                } else if (mSkip == "&quot;" && ch == ';') {
+                } else if (mSkip == "&quot;" ) {
```

Comment:
:beetle: :stop_sign: I meant take the `;` out of the `mSkip == "&quot;"` part - not to take out the check for it in the *current* character in `ch`. :man_facepalming:

Reply:
Aha that makes more sense. I missed it in the morning...

Reply:
so it should be `&quot` instead of `&quot;` -- correct? :thought_balloon:

Reply:
Yeah